CMakeLists.txt

```diff
@@ -4,19 +4,13 @@
 
 cmake_minimum_required(VERSION 3.5.1)
 
-include(evmc/cmake/cable/bootstrap.cmake)
+include(cmake/cable/bootstrap.cmake)
 include(CableBuildInfo)
 include(CableBuildType)
 include(CableToolchains)
 include(GNUInstallDirs)
 
-if(UNIX)
-    # Build deps with PIC to allow building shared libraries.
-    set(toolchain cxx11-pic)
-else()
-    set(toolchain cxx11)
-endif()
-cable_configure_toolchain(DEFAULT ${toolchain})
+cable_configure_toolchain(DEFAULT cxx11-pic)
```

Comment:
`fpic`?

Reply:
No, I'm testing the simpler one because Hunter said it might work.

Reply:
Seems to be working nicely, so `fpic` variant is not needed.